sysutils/snapraid/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        amadvance snapraid 12.3 v
+
+name                snapraid
+version             12.3
+categories          sysutils
+
+license             GPL-3
+maintainers         {anquieta @anquieta}
+description         Backup program for disk arrays
+long_description    SnapRAID stores parity information of your data and \
+                    it recovers from up to six disk failures
+homepage            http://www.snapraid.it/
+
+github.tarball_from downloads
+
+distname            ${github.project}-${github.version}
+
+master_sites        ${github.homepage}/releases/download/${github.tag_prefix}${github.version}/
```

Comment:
`master_sites` should not need to be set if you're using the `github` portgroup.  You should be able to delete this line.